CMakeLists.txt

```diff
@@ -801,7 +801,7 @@ set(OPENSIM_SYSTEM_INFO ${CMAKE_SYSTEM})
 set(OPENSIM_OS_NAME ${CMAKE_SYSTEM_NAME})
 
 if( WIN32 )
-    if (${MSVC_VERSION} GREATER 1910)
+    if (NOT (MSVC_VERSION LESS 2000))
```

Comment:
why use `NOT ... LESS` opposed to `GREATER  2000`

Reply:
The warning should be generated if MSCVERSION == 2000. I would prefer to use `GREATER_EQUAL` but [this was only introduced in CMake 3.7](https://stackoverflow.com/questions/16667017/cmake-express-the-greater-or-equal-statement) and our minimum required CMake version is 3.2.